### DISCUSSION TITLE
Problem with Reactions field

### DISCUSSION THREAD

**Discussion by StriXus17** [kevinzg/facebook-scraper#743] [General]
Hi !
the reactions field keeps returning nan values, I read in the docs about ` extra_info` I used it in the `get_posts` function but it doesn't sem to fix anything, am I missing something ?
This is how I wrote my function:  `get_posts("nintendo", pages=4,options={'comments': True,'reactions':True},extra_info=True)`
Thanks in advance .

**Comment by neon-ninja** [comment]
Try pass cookies as per the readme

**Reply by StriXus17** [reply]
thanks a lot, it works now ! even though it takes much more time but at least it works.
